DRAFT.md

```diff
@@ -821,21 +817,15 @@ explicit constexpr indirect(allocator_arg_t, const Allocator& a, in_place_t, Us&
     owned object of type `T` with `std​::​forward<Us>(us)...`, using the
     allocator `alloc`.
 
-19. _Throws_: Nothing unless `allocator_traits<allocator_type>::allocate` or
-  `allocator_traits<allocator_type>::construct` throws.
-
 ```c++
 constexpr indirect(const indirect& other);
 ```
 
-20. _Effects_: Equivalent to `indirect(allocator_arg_t{},
-    allocator_traits<allocator_type>::select_on_container_copy_construction(other.alloc),
-    other)`
+??. _Mandates_: `T` is a complete type.
```

Comment:
Why `"??"` here, was this a temporary reminder to update the number?  This should be updated to the correct paragraph number.

Reply:
Thanks Ant. This had escaped me. Will fix and update.